altair/api.py

```diff
@@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
             self.shape._infer_type(self.parent.data)
 
 
+class Filter():
```

Comment:
Right after we release 0.8, we will start drafting syntax for composition (especially layer) so we can have a standard even before we get to implement them.